src/test/java/org/opencds/cqf/tooling/utilities/ECQMCreatorIT.java

```diff
@@ -67,1278 +68,427 @@ private String measureToString(Measure measure) {
-            logger.debug(measureToString(measure));
+            logger.debug(measString);
         }
         catch (IOException ioException) {
             ioException.printStackTrace();
         }
+
+        return drs;
     }
-    
-    @Test
-    public void TestCMS130FHIR() {
-        // Extract the bundle
-        // NOTE: This is a 2021-AUFHIR measure, this is the test created using TestCMS125FHIR as template
-        ExtractMatBundleOperation o = new ExtractMatBundleOperation();
-        o.execute(new String[] { "-ExtractMATBundle", this.getClass().getResource("ecqm-content-r4-2021/bundles/CMS130FHIR-v0-0-002-FHIR-4-0-1.json").getFile() });
 
-        try {
-            Measure measure = refreshMeasure("ecqm-content-r4-2021/input/cql/ColorectalCancerScreeningsFHIR.cql", "ecqm-content-r4-2021/input/resources/measure/ColorectalCancerScreeningsFHIR.json");
-            assertTrue(null != measure);
-            // Extract data requirements from the measure:
-            List<DataRequirement> drs = new ArrayList<DataRequirement>();
-            for (Extension e : measure.getExtensionsByUrl("http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-dataRequirement")) {
-                if (e.hasValue()) {
-                    drs.add(e.getValueDataRequirement());
+    private void checkExpectedResourcesPresent(List<DataRequirement> drs, List<String> edrs){
+        boolean dataRequirementFound;
+        for (String edr : edrs){
+            dataRequirementFound = false;
+            for (DataRequirement dr : drs){
+                if (edr.equalsIgnoreCase(dr.getType().toString())){
+                    dataRequirementFound=true;
                 }
             }
-            assertTrue(!drs.isEmpty());
-            // TODO: Measure-specific validation of data requirements content
-            logger.debug(measureToString(measure));
-        }
-        catch (IOException ioException) {
-            ioException.printStackTrace();
+            if (dataRequirementFound){
+                System.out.println("found expected data requirement " + edr );
+            }else{
+                System.out.println("unable to find expected data requirement " + edr );
+                assertTrue(false);
+            }
         }
     }
+
+
+    @Test
+    public void TestCMS125FHIR() {
+        List<DataRequirement> drs = StartMatOutputTest("CMS125FHIR-v0-0-004-FHIR-4-0-1.json", "BreastCancerScreeningsFHIR");
+        // TODO: Measure-specific validation of data requirements content
+    }
+
+    @Test
+    public void TestCMS104FHIR() {
+        List<DataRequirement> drs = StartMatOutputTest("CMS104-v2-0-004-FHIR-4-0-1.json", "DischargedonAntithromboticTherapyFHIR");
+        // TODO: Measure-specific validation of data requirements content
+
+        List<String> edrs = List.of("ServiceRequest","Procedure","Encounter");
+        checkExpectedResourcesPresent(drs, edrs);
+
+    }
```

Comment:
This is a simple test I added to CMS 104 to check to make sure that the expected FHIR resources are present. I am not sure that this list is complete, but this is enough for the test to fail, because the expected resource "Encouter" is not present